### PR TITLE
webui: Add ability to mark all notifications as read

### PR DESCRIPTION
**Pull request by Rosiak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

fixes:
https://community.librenms.org/t/ability-to-mark-all-notifications-as-re
ad/2368/1

However it's done in a slightly ugly way with the window.location.reload() as my JS skills are lacking, and it's late. So please give some input :)

### CODE REVIEW

File: html/pages/notifications.inc.php [hunk 1]
```diff
@@ -30,7 +30,8 @@
   <div class="row">
     <div class="col-md-12">
       <h1><a href="/notifications">Notifications</a></h1>
-      <h4><strong class="count-notif"><?php echo $notifications['count']; ?></strong> Unread Notifications <?php echo ($_SESSION['userlevel'] == 10 ? '<button class="btn btn-success pull-right new-notif" style="margin-top:-10px;">New</button>' : ''); ?></h4>
+      <h4><strong class="count-notif"><?php echo $notifications['count']; ?></strong> Unread Notifications <?php echo ($_SESSION['userlevel'] == 10 ? '<button class="btn btn-success pull-right fa fa-plus new-notif" data-toggle="tooltip" data-placement="bottom" title="Create new notification" style="margin-top:-10px;"></button>' : ''); ?><?php echo ($_SESSION['userlevel'] == 10 ? '<button class="btn btn-success pull-right fa fa-eye read-all-notif" data-toggle="tooltip" data-placement="bottom" title="Mark all as Read" style="margin-top:-10px;"></button>' : ''); ?>
```
Comment (murrant) [inline html/pages/notifications.inc.php]: Can you use is_admin() instead of checking $_SESSION directly

**Comment by murrant** [comment]
Found an issue, if you keep clicking the button, it keeps adding notifications_attribs entries.

Also, why is this admin only?

**Comment by laf** [comment]
@Rosiak ?

**Comment by Rosiak** [comment]
Seems like this should be possible for every user, I agree on that.
That said I haven't had the time to look further into it - I'll try to see if I have a change in the coming week or the week after.

File: html/pages/notifications.inc.php [hunk 2]
```diff
@@ -232,6 +233,24 @@
...
+  $(document).on( "click", ".read-all-notif", function() {
+    $(this).attr("disabled", true);
+    $.ajax({
+      type: 'POST',
+      url: 'ajax_form.php',
+      data: {type: 'notifications', action: 'read-all-notif'},
+      dataType: "json",
+      success: function (data) {
+        if( data.status == "ok" ) {
+          window.location.reload()
```
Comment (murrant) [inline html/pages/notifications.inc.php]: Should be able to hide all unread notifications and update the notification counts in jquery.

File: html/pages/notifications.inc.php (hunk 1 above)
Comment (murrant) [inline html/pages/notifications.inc.php]: Mark All as read should not be displayed on the notifications page.

**Pull request merged**